src/modules/homepage/index.tsx

```diff
@@ -0,0 +1,430 @@
+              justifyContent='center'
+              marginLeft='16px'
+            >
+              <ArrowUpwardIcon />
+              <CameraAltIcon />
+            </Box>
+            <Box color='#fff' marginLeft='8px'>
+              <Typography sx={{ fontSize: '14px' }}>Expenses</Typography>
+              <Typography sx={{ fontSize: '22px', fontWeight: '600' }}>
+                $1200
+              </Typography>
+            </Box>
+          </Box>
+        </Box>
+
+        {/* <StyledPageContentDiv>
+                    <div>
+                        <Link to='/onboarding/introduction'>
+                            Onboarding introduction
+                        </Link>
+                    </div>
+                    <div>
+                        <Link to='/expense/new'>Add new expense</Link>
+                    </div>
+                </StyledPageContentDiv> */}
+      </StyledRectangle>
+
+      <Box padding='8px'>
+        <Typography
+          width='375px'
+          height='48px'
+          fontSize='18px'
+          fontWeight='600'
+          variant='h6'
+          lineHeight='21.78px'
+          textAlign='left'
+        >
+          Spend Frequency
+        </Typography>
+      </Box>
+
+      <Box width='375px' height='185.5px'>
+        <Graphic />
+      </Box>
+
+      <Box display='flex' flexDirection='row'>
+        <Typography
+          fontWeight='500'
+          fontSize='14px'
+          lineHeight='18px'
+          textAlign='center'
+          padding='8px 24px'
+          color='#FCAC12'
+          bgcolor='#FCEED4'
+          borderRadius='16px'
+        >
+          Today
+        </Typography>
+        <Typography
+          fontWeight='500'
+          fontSize='14px'
+          lineHeight='18px'
+          textAlign='center'
+          padding='8px 24px'
+          color='#91919F'
+        >
+          Week
+        </Typography>
+        <Typography
+          fontWeight='500'
+          fontSize='14px'
+          lineHeight='18px'
+          textAlign='center'
+          padding='8px 24px'
+          color='#91919F'
+        >
+          Month
+        </Typography>
+        <Typography
+          fontWeight='500'
+          fontSize='14px'
+          lineHeight='18px'
+          textAlign='center'
+          padding='8px 24px'
+          color='#91919F'
+        >
+          Year
+        </Typography>
+      </Box>
+
+      <Box
+        padding='8px'
+        display='flex'
+        flexDirection='row'
+        alignItems='center'
+        justifyContent='space-between'
+      >
+        <Typography fontWeight='600' fontSize='18px' lineHeight='21.78px'>
+          Recent Transaction
+        </Typography>
+        <Typography
+          borderRadius='40px'
+          padding='8px 16px'
+          bgcolor='#EEE5FF'
+          color='#7F3DFF'
+          fontSize='14px'
+        >
+          See All
+        </Typography>
+      </Box>
+
+      <Box display='flex' flexDirection='row' padding='15px'>
```

Comment:
This should be a shared component. There's a transaction card component: https://www.figma.com/design/6KSlLFezGkD8uZXhuTsx3j/Montra---Expense-Tracker-UI-Kit-(Community)?node-id=642-56&t=owVDwKh8pLVF3EUv-0

Reply:
I created shared component named TransactionCard with props. But on the figma there is a condition for TransactionCard which is expense=no or expense=yes so the price color changing depending on this condition. I did not add this since we adding values hard coding for now.